adding_species_manuscript.tex

```diff
@@ -634,114 +641,112 @@ \subsection*{Extracting parameters from the literature}
+		\\
+		\hline
+		Gene conversion rate and tract length &
+		Set rate to 0 or borrow from closest relative with a citeable rate &
+		Lengths of shared haplotypes across individuals
+		\\
+		\hline
+		Demographic model &
+		Set the effective population size (Ne) to a value
+		that reflects the average observed genetic diversity in the
+		simulated population
+		     &
+		Features of genetic diversity that are captured by the site frequency spectrum,
+		such as the prevalence of low-frequency alleles\\
+		\hline
+	\end{tabular}
+\end{table}
+
+%ILAN: following suggestions from others, I moved most of this into the table,
+% and added a clearer description of the table in the main text.
+For many species it is difficult to obtain estimates of all necessary model parameters.
+Table \ref{tab:param-mod} provide suggestions for ways to deal with missing values of various central model parameters.
+The table also mentions the main discrepancies between the simulated data and real genomic data,
+which can be caused by mis-specification of each parameter.
+
+%ILAN: shortened the paragraph by removing a lot of the low-level considerations,
+%  following a suggestion by Jerome
+Several researchers who participated in the ``Growing the Zoo'' hackathon wished to add species
+whose genome assemblies are composed of many relatively small contigs,
+unanchored to chromosome-level scaffolds.
+Although we wish to keep \stdpopsim as inclusive as possible,
+we made a conscience decision to only add species with chromosome-level assemblies.
+%
+The main justification for this restriction is that
+species with less complete genome builds typically do not have good estimates of recombination rate, genetic maps, and demographic models,
+making chromosome-level simulation much less useful in such species.
+%
+Another issue is the storage burden and long load times involved in dealing with
+hundreds of contigs.
+%
+Finally, each species requires validation of its code before it is added to the \stdpopsim catalog,
+as well as long-term maintenance to keep it up-to-date after changes to the \stdpopsim framework.
+So, the benefit of including species with very partial genome builds in \stdpopsim
+would be outweighed by the substantial extra burden on \stdpopsim maintainers as well as
+downstream users of these models.
+
+ 
+That being said, simulation is still possible and potentially useful for species with partial genome builds.
 One way to deal with this situation is to include only the longer contigs or scaffolds,
 treating them as separate chromosomes in the simulation.
 Some of these contigs will map to the same chromosome, 
 so simulating them separately will not capture the genetic linkage between them.
 However, this provides a reasonable approximation for many purposes, at least for genomic regions far from the contig edges.
 Short contigs can either be omitted from simulation, or lumped together into one (or several) longer pseudo-chromosome(s).
```

Comment:
I think the deleted stuff in the rest of this paragraph is useful, and it's part of the point of this paper to explain the underlying tricky issues.

Reply:
The point about "false precision" is that if someone thinks "oh, I'm precisely simulating segment 18374499 to 4885722 of chromsome 18" then they're missing the point that well yes the features on that segment might be 'precise', but the linked effects of other bits of chromosome 18 are missing.

Reply:
I'm still not quite sure what the point of 'false precision' is. I'm going to try to provide more in this statement by modifying it to:

> Creating pseudo-chromosomes allows the simulation to fit the amount of data of real genomes **and capture the linkage structure within each contig, but it induces false or inaccurate correlation between variants from different contigs.**
(in bold the modified part of the sentence)

Reply:
okay, here's the point: suppose that someone wants to simulate a genome that is just in a lot of contigs; and they put together the code to run neutral simulations of lots of unlinked chunks of exactly the same size as their contigs, so that each bit of real data is carefully matched to a simulated chunk. Although this seems very precise and accurate somehow, there is (almost) no point at all to doing this careful matching up. The advantages to realistic matching up of chromosomes are, I think mostly: (1) realistic recombination maps and (2) realistic varying strengths of selection (with exon density); but you get neither advantage from lots of anonymous unlinked contigs.

This is not a hypothetical situation, and is tricky and something we spent a good deal of time talking about, which is why I'm trying to explain it.

Reply:
I'm starting to get the point (finally). So, I plan to add a thoughtful discussion about the benefits of lumping contigs into pseudo chromosomes (mentioning the obvious caveats), using the arguments you make in your comments (here and below). You've now got me on board on the position that the benefits outweigh the negative effects :-). If I'm not mistaken, the same points apply here (when we discuss a build that has long and short contigs) and in the case we mention later (with only short contigs). So, I think it makes sense to address this once and not as two separate issues. I will also remove the mention of the option to omit the short contigs. That's always an option I guess, and it distracts from the main point here. Does this plan make sense, @petrelharp ?

Reply:
BTW, we might not want to make strict distinction between short and long contigs here. Clearly, lumping makes more sense for shorter contigs, but it might make more sense to blur the line here a bit.